pkg/test/framework/settings/flags.go

```diff
@@ -86,38 +64,6 @@ func processFlags() error {
 	return nil
 }
 
-func attachFlags(stringVar func(*string, string, string, string), boolVar func(*bool, string, bool, string)) {
-	stringVar(&settings.WorkDir, "work_dir", os.TempDir(),
-		"Local working directory for creating logs/temp files. If left empty, os.TempDir() is used.")
-
-	stringVar((*string)(&settings.Environment), "environment", string(settings.Environment),
-		fmt.Sprintf("Specify the environment to run the tests against. Allowed values are: [%s, %s]",
-			Local, Kubernetes))
-
-	stringVar(&settings.KubeConfig, "config", settings.KubeConfig,
-		"The path to the kube config file for cluster environments")
-
-	boolVar(&showHelp, "hh", showHelp,
-		"Show the help page for the test framework")
-
-	boolVar(&settings.NoCleanup, "no-cleanup", settings.NoCleanup, "Do not cleanup resources after test completion")
-}
-
-func doShowHelp() {
```

Comment:
Does regular help page for go test show our flags? If not, I think we should hold on to the -hh
flag.

Reply:
looks like `--h` just works out of the box.  I get this with the `-h` option:

```
Usage of /tmp/___TestHTTP_in_istio_io_istio_tests_integration2_examples_pilot:
  -config string
    	The path to the kube config file for cluster environments
  -environment string
    	Specify the environment to run the tests against. Allowed values are: [local, kubernetes] (default "local")
  -no-cleanup
    	Do not cleanup resources after test completion
  -test.bench regexp
    	run only benchmarks matching regexp
  -test.benchmem
    	print memory allocations for benchmarks
  -test.benchtime d
    	run each benchmark for duration d (default 1s)
  -test.blockprofile file
    	write a goroutine blocking profile to file
  -test.blockprofilerate rate
    	set blocking profile rate (see runtime.SetBlockProfileRate) (default 1)
  -test.count n
    	run tests and benchmarks n times (default 1)
  -test.coverprofile file
    	write a coverage profile to file
  -test.cpu list
    	comma-separated list of cpu counts to run each test with
  -test.cpuprofile file
    	write a cpu profile to file
  -test.failfast
    	do not start new tests after the first test failure
  -test.list regexp
    	list tests, examples, and benchmarks matching regexp then exit
  -test.memprofile file
    	write an allocation profile to file
  -test.memprofilerate rate
    	set memory allocation profiling rate (see runtime.MemProfileRate)
  -test.mutexprofile string
    	write a mutex contention profile to the named file after execution
  -test.mutexprofilefraction int
    	if >= 0, calls runtime.SetMutexProfileFraction() (default 1)
  -test.outputdir dir
    	write profiles to dir
  -test.parallel n
    	run at most n tests in parallel (default 12)
  -test.run regexp
    	run only tests and examples matching regexp
  -test.short
    	run smaller test suite to save time
  -test.testlogfile file
    	write test action log to file (for use only by cmd/go)
  -test.timeout d
    	panic test binary after duration d (default 0, timeout disabled)
  -test.trace file
    	write an execution trace to file
  -test.v
    	verbose: print additional output
  -work_dir string
    	Local working directory for creating logs/temp files. If left empty, os.TempDir() is used. (default "/tmp")
```